js/app.js

```diff
@@ -1,2 +1,44 @@
 $(function () {
+  $("#search").submit(() =>{
+    event.preventDefault()
+
+
+    console.log("submitting form")
+    // read value from text field
+    //store in query variable
+    const query = $("#query").val()
+    console.log(query)
+    search(query)
+
+  })
+
+  function search(term){
+    //this is where we make our api request
+    console.log(`calling search with q=${term}`)
+    const url = "https://api.openweathermap.org/data/2.5/weather"
+    const apiKey = "b1fc5e5396249435c53ad57e7f01f7bf"
+
+    jQuery.ajax({
```

Comment:
@cbrooks2019 nice job setting up your `.ajax()` call. While this should work, best practice is to use the shorthand syntax (`$`) instead of `jQuery`